b1ddi/schema_dns_data_record_options.go

```diff
@@ -0,0 +1,32 @@
+package b1ddi
+
+import (
+	"terraform-provider-b1ddi/b1ddi/util"
+)
+
+func deepUpdateDataRecordOptions(d interface{}, recordType string) (interface{}, error) {
```

Comment:
Could you add the comment here, or change the function name. It's not clear what `deepUpdateDataRecordOptions` means.

Reply:
updated